db/models.py

```diff
@@ -19,8 +19,8 @@ def __str__(self) -> str:
 class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
-    actors = models.ManyToManyField(Actor, related_name="movie_actors")
-    genres = models.ManyToManyField(Genre, related_name="movie_genres")
+    actors = models.ManyToManyField(Actor, related_name="actors_of_movies")
+    genres = models.ManyToManyField(Genre, related_name="genres_of_movies")
```

Comment:
imagine how would u call it using orm 
`Actor.objects.get(<some_id>).actors_of_movies` to get all related movies to an actor 
maybe it is better to use `movies` here ?